### ISSUE TITLE
Add test for missed functions

### ISSUE THREAD

**Issue by Alexandre-T** [longitude-one/doctrine-spatial#11]
There is no test for the next files: 
* PostGis: [SpScale.php](https://coveralls.io/builds/40810010/source?filename=lib%2FLongitudeOne%2FSpatial%2FORM%2FQuery%2FAST%2FFunctions%2FPostgreSql%2FSpScale.php)
* PostGis: [SpTransform.php](https://coveralls.io/builds/40810010/source?filename=lib%2FLongitudeOne%2FSpatial%2FORM%2FQuery%2FAST%2FFunctions%2FPostgreSql%2FSpTransform.php)
* PostGis: [SpTranslate.php](https://coveralls.io/builds/40810010/source?filename=lib%2FLongitudeOne%2FSpatial%2FORM%2FQuery%2FAST%2FFunctions%2FPostgreSql%2FSpTranslate.php)
* Standard: [StConvexHull.php](https://coveralls.io/builds/40810010/source?filename=lib%2FLongitudeOne%2FSpatial%2FORM%2FQuery%2FAST%2FFunctions%2FStandard%2FStConvexHull.php)

**Comment by lacatoire** [comment]
This issue could be closed ? @Alexandre-T 

- [SpScaleTest](https://github.com/longitude-one/doctrine-spatial/blob/main/tests/LongitudeOne/Spatial/Tests/ORM/Query/AST/Functions/PostgreSql/SpScaleTest.php)
- [SpTransformTest](https://github.com/longitude-one/doctrine-spatial/blob/main/tests/LongitudeOne/Spatial/Tests/ORM/Query/AST/Functions/PostgreSql/SpTransformTest.php)
- [SpTranslateTest](https://github.com/longitude-one/doctrine-spatial/blob/main/tests/LongitudeOne/Spatial/Tests/ORM/Query/AST/Functions/PostgreSql/SpTranslateTest.php)
- [StConvexHullTest](https://github.com/longitude-one/doctrine-spatial/blob/main/tests/LongitudeOne/Spatial/Tests/ORM/Query/AST/Functions/Standard/StConvexHullTest.php)

**Comment by Alexandre-T** [comment]
No, these functions aren't fully tested. As you can read it, [tests are skipped](https://github.com/longitude-one/doctrine-spatial/blob/3978cca3a034086de199a0c47d63742ac7e5fd22/tests/LongitudeOne/Spatial/Tests/ORM/Query/AST/Functions/PostgreSql/SpScaleTest.php#L74)
